client-cli/src/test/java/org/eclipse/dataspaceconnector/registration/cli/ParticipantsCommandTest.java

```diff
@@ -84,6 +84,22 @@ void add() {
         assertAddParticipants(exitCode, dataspaceDid, app.dataspaceDid);
     }
 
+    @Test
+    void status() throws Exception {
+        when(app.registryApiClient.getParticipantStatus())
+                .thenReturn(participant1);
+
+        var exitCode = executeParticipantStatus();
+        assertThat(exitCode).isEqualTo(0);
+
+        var parsedResult = MAPPER.readValue(sw.toString(), new TypeReference<ParticipantDto>() {
```

Comment:
```suggestion
        var parsedResult = MAPPER.readValue(sw.toString(), ParticipantDto.class)
```

`TypeReference` is only required for generic types

Reply:
Applied. Thanks